docs/css/styles.css

```diff
@@ -133,26 +133,33 @@
 
```

Comment:
This looks like compiled scss. Just curious, is there a reason for committing compiled assets to the repo?